<?xml version="1.0" encoding="UTF-8"?><testrun name="TestConector" project="jrpg-2017b-servidor" tests="10" started="10" failures="0" errors="2" ignored="0">
  <testsuite name="testsServidor.TestConector" time="5.884">
    <testcase name="testConexionConLaDB" classname="testsServidor.TestConector" time="0.732"/>
    <testcase name="registrarPersonaje" classname="testsServidor.TestConector" time="2.921"/>
    <testcase name="testRegistrarUsuario" classname="testsServidor.TestConector" time="0.015"/>
    <testcase name="asignarMochila" classname="testsServidor.TestConector" time="0.547">
      <error>javax.persistence.PersistenceException: org.hibernate.exception.ConstraintViolationException: could not execute statement&#13;
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:149)&#13;
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:157)&#13;
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:164)&#13;
	at org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1443)&#13;
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:493)&#13;
	at org.hibernate.internal.SessionImpl.flushBeforeTransactionCompletion(SessionImpl.java:3207)&#13;
	at org.hibernate.internal.SessionImpl.beforeTransactionCompletion(SessionImpl.java:2413)&#13;
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.beforeTransactionCompletion(JdbcCoordinatorImpl.java:467)&#13;
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.beforeCompletionCallback(JdbcResourceLocalTransactionCoordinatorImpl.java:156)&#13;
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.access$100(JdbcResourceLocalTransactionCoordinatorImpl.java:38)&#13;
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:231)&#13;
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:68)&#13;
	at entidades.EntMochila.asignarMochila(EntMochila.java:69)&#13;
	at testsServidor.TestConector.asignarMochila(TestConector.java:171)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute statement&#13;
	at com.enigmabridge.hibernate.dialect.SQLiteDialect$2.convert(SQLiteDialect.java:248)&#13;
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)&#13;
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:111)&#13;
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:97)&#13;
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:178)&#13;
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:45)&#13;
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3013)&#13;
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3513)&#13;
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:89)&#13;
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:589)&#13;
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:463)&#13;
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:337)&#13;
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:39)&#13;
	at org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1437)&#13;
	... 33 more&#13;
Caused by: org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: mochila.idMochila)&#13;
	at org.sqlite.core.DB.newSQLException(DB.java:909)&#13;
	at org.sqlite.core.DB.newSQLException(DB.java:921)&#13;
	at org.sqlite.core.DB.execute(DB.java:825)&#13;
	at org.sqlite.core.DB.executeUpdate(DB.java:863)&#13;
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)&#13;
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:175)&#13;
	... 42 more&#13;
&#13;
</error>
    </testcase>
    <testcase name="testLoginUsuarioFallido" classname="testsServidor.TestConector" time="0.026"/>
    <testcase name="probarEntidadRegistro" classname="testsServidor.TestConector" time="0.555"/>
    <testcase name="testLoginUsuario" classname="testsServidor.TestConector" time="0.016"/>
    <testcase name="asignarInventario" classname="testsServidor.TestConector" time="0.002"/>
    <testcase name="testRegistrarPersonaje" classname="testsServidor.TestConector" time="0.536"/>
    <testcase name="levantarItems" classname="testsServidor.TestConector" time="0.534">
      <error>javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: could not prepare statement&#13;
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:149)&#13;
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:157)&#13;
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1423)&#13;
	at entidades.EntItem.levantarItems(EntItem.java:34)&#13;
	at testsServidor.TestConector.levantarItems(TestConector.java:176)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: org.hibernate.exception.GenericJDBCException: could not prepare statement&#13;
	at com.enigmabridge.hibernate.dialect.SQLiteDialect$2.convert(SQLiteDialect.java:256)&#13;
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)&#13;
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:111)&#13;
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:182)&#13;
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:148)&#13;
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1995)&#13;
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1925)&#13;
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1903)&#13;
	at org.hibernate.loader.Loader.doQuery(Loader.java:948)&#13;
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:351)&#13;
	at org.hibernate.loader.Loader.doList(Loader.java:2702)&#13;
	at org.hibernate.loader.Loader.doList(Loader.java:2685)&#13;
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2517)&#13;
	at org.hibernate.loader.Loader.list(Loader.java:2512)&#13;
	at org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:502)&#13;
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:384)&#13;
	at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:216)&#13;
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1490)&#13;
	at org.hibernate.query.internal.AbstractProducedQuery.doList(AbstractProducedQuery.java:1445)&#13;
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1414)&#13;
	... 25 more&#13;
Caused by: org.sqlite.SQLiteException: [SQLITE_ERROR] SQL error or missing database (no such column: entitem0_.wareable)&#13;
	at org.sqlite.core.DB.newSQLException(DB.java:909)&#13;
	at org.sqlite.core.DB.newSQLException(DB.java:921)&#13;
	at org.sqlite.core.DB.throwex(DB.java:886)&#13;
	at org.sqlite.core.NativeDB.prepare_utf8(Native Method)&#13;
	at org.sqlite.core.NativeDB.prepare(NativeDB.java:127)&#13;
	at org.sqlite.core.DB.prepare(DB.java:227)&#13;
	at org.sqlite.core.CorePreparedStatement.&lt;init&gt;(CorePreparedStatement.java:41)&#13;
	at org.sqlite.jdbc3.JDBC3PreparedStatement.&lt;init&gt;(JDBC3PreparedStatement.java:30)&#13;
	at org.sqlite.jdbc4.JDBC4PreparedStatement.&lt;init&gt;(JDBC4PreparedStatement.java:19)&#13;
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:48)&#13;
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:254)&#13;
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:226)&#13;
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:146)&#13;
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:172)&#13;
	... 41 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
